pkgs/applications/graphics/gimp/plugins/default.nix

```diff
@@ -66,6 +66,36 @@ in
   # Allow overriding GIMP package in the scope.
   inherit (pkgs) gimp;
 
+  bimp = pluginDerivation rec {
+    /* menu:
+       File/Batch Image Manipulation...
+    */
+    pname = "bimp";
+    version = "2.6";
+
+    src = fetchFromGitHub {
+      owner = "alessandrofrancesconi";
+      repo = "gimp-plugin-bimp";
+      rev = "v${version}";
+      hash = "sha256-IJ3+/9UwxJTRo0hUdzlOndOHwso1wGv7Q4UuhbsFkco=";
+    };
+
+    nativeBuildInputs = with pkgs; [ which ];
+
+    installPhase = ''
+      runHook preInstall
+      installPlugin bin/bimp
```

Comment:
This will not install translations and stuff. I would prefer something like this:

```patch
--- a/pkgs/applications/graphics/gimp/plugins/default.nix
+++ b/pkgs/applications/graphics/gimp/plugins/default.nix
@@ -5,7 +5,7 @@
 { config, lib, pkgs }:
 
 let
-  inherit (pkgs) stdenv fetchurl pkg-config intltool glib fetchFromGitHub;
+  inherit (pkgs) stdenv fetchurl fetchpatch pkg-config intltool glib fetchFromGitHub;
 in
 
 lib.makeScope pkgs.newScope (self:
@@ -80,13 +80,22 @@ in
       hash = "sha256-IJ3+/9UwxJTRo0hUdzlOndOHwso1wGv7Q4UuhbsFkco=";
     };
 
+    patches = [
+      # Allow overriding installation path
+      # https://github.com/alessandrofrancesconi/gimp-plugin-bimp/pull/311
+      (fetchpatch {
+        url = "https://github.com/alessandrofrancesconi/gimp-plugin-bimp/commit/098edb5f70a151a3f377478fd6e0d08ed56b8ef7.patch";
+        sha256 = "2Afx9fmdn6ztbsll2f2j7mfffMWYWyr4BuBy9ySV6vM=";
+      })
+    ];
+
     nativeBuildInputs = with pkgs; [ which ];
 
-    installPhase = ''
-      runHook preInstall
-      installPlugin bin/bimp
-      runHook postInstall
-    '';
+    installFlags = [
+      "SYSTEM_INSTALL_DIR=${placeholder "out"}/${gimp.targetPluginDir}/bimp"
+    ];
+
+    installTargets = [ "install-admin" ];
 
     meta = with lib; {
       description = "Batch Image Manipulation Plugin for GIMP";
```

Reply:
@jtojnar Good catch, thanks for the patch!